src/manifold/src/impl.h

```diff
@@ -66,6 +66,29 @@ struct Manifold::Impl {
        const std::vector<float>& propertyTolerance = {},
        bool hasFaceIDs = false);
 
+  inline void ForVert(int halfedge, std::function<void(int halfedge)> func) {
+    int current = halfedge;
+    do {
+      func(current);
+      current = NextHalfedge(halfedge_[current].pairedHalfedge);
+    } while (current != halfedge);
+  }
+
+  template <typename T>
+  void ForVert(int halfedge, std::function<T(int halfedge)> transform,
+               std::function<void(int halfedge, const T& here, const T& next)>
+                   binaryOp) {
```

Comment:
@pca006132 Don't worry about a serious review - I was mostly curious if you thought this was a reasonable way to handle #775.